packages/nft-checkout/src/scripts/generate-liqudity-pairs-avalanche.js

```diff
@@ -0,0 +1,76 @@
+const fetcher = require('@traderjoe-xyz/sdk')
+const { Fetcher } = require('@distributedlab/fetcher')
+const { Token: TJToken } = require("@traderjoe-xyz/sdk/dist");
+const { ethers } = require("ethers")
+const fs = require('fs');
+const DEFAULT_FETCHER_CONFIG = {
+  credentials: 'omit',
+  referrerPolicy: 'no-referrer',
+}
+const FUJI_CHAIN = 43113
+const AVALANCHE_CHAIN = 43114
+const AVALANCHE_RPC_URL = 'https://avalanche.public-rpc.com'
+const FUJI_RPC_URL = 'https://rpc.ankr.com/avalanche_fuji'
+
+const getProvider = (chain) => chain === FUJI_CHAIN ? new ethers.providers.JsonRpcProvider(FUJI_RPC_URL) :
+  new ethers.providers.JsonRpcProvider(AVALANCHE_RPC_URL)
+const generatePairs = async (tokenListUrls) => {
+  let tokenLists= []
+
+  for (const url of tokenListUrls) {
+    try {
+      const response = async (rawUrl) => {
+        const url = new URL(rawUrl)
+
+        const fetcher = new Fetcher({
+          baseUrl: url.origin,
+          ...DEFAULT_FETCHER_CONFIG,
+        })
+        return fetcher.get(url.pathname)
+      }
```

Comment:
move this fn outside for, it causes performance issues

Reply:
fixed